cluster/gce/gci/configure.sh

```diff
@@ -26,7 +26,7 @@ set -o pipefail
 ### Hardcoded constants
 DEFAULT_CNI_VERSION='v1.4.1'
 DEFAULT_CNI_HASH='5cff10005288a78b484bbabbaa24669d62a6eedb6cc7c7604b2c1ea14b65e90a1b022c2a2975d7764ec41f24707f6349432e8f11564d03cea0da141195b0602e'
-DEFAULT_NPD_VERSION='v0.8.16'
+DEFAULT_NPD_VERSION='v0.8.19'
```

Comment:
don't we need to update the has? does it work without it?

Reply:
atleast tests are not failing. are these hashes in effect only when `tag` is not present ?

Reply:
yeah, I meant hash sorry, I can see those were bumped last time https://github.com/kubernetes/kubernetes/commit/9ac9fdac8f8d7a0a6b0f34a22b3ae0f00d7559ee

> DEFAULT_NPD_HASH_AMD64
> DEFAULT_NPD_HASH_ARM64

Reply:
for some reason I failed to get the exact command to retrieve the hashes as in this file and don't see documentation on the same. @dims do you know the method/command to get the same?

Reply:
This part of the script runs in NPD in standalone mode. So it runs NPD as a system daemon instead of a pod. So it downloads the tar file from github release nstead of an image and checks that sha512/sha1 of the tar file is the same as mentioned in this env.

For v0.8.19, it is:

```
DEFAULT_NPD_HASH_AMD64='41e7816614a1e30c94cbe37a1d09c2ef6c3f2ad336b40d95378528deac9a8b811356e46bae268dab006132f2bdce22668a640517f000853109d1e3be77aef25e'
DEFAULT_NPD_HASH_ARM64='678352d0544c166e86d3435c4f49b1ea5ce19c2d6f083f51f7688599377928e5b1425d1733006981d17e94ddede67f9fc1d2035577a87687701044950f697390'
```

I also did not find any presubmit (or in fact *any*) tests covering these lines of the script. In fact, most tests are either not running NPD or running it as a daemonset. It would be good to add it.

Reply:
That answers it. I have updated the hash in the PR. ptal . thanks .